test/test_bson.py

```diff
@@ -1053,6 +1054,57 @@ def test_unicode_decode_error_handler(self):
         self.assertRaises(InvalidBSON, decode, invalid_both, CodecOptions(
             unicode_decode_error_handler="junk"))
 
+    def round_trip_pickle(self, object, pickled_with_older=None):
+        if pickled_with_older is not None:
+            pickled_with_older = pickle.loads(pickled_with_older)
```

Comment:
Changing the type of `pickled_with_older` from `bytes` to an object makes this could more difficult to understand. Please make this a new variable rather than reassigning.

Reply:
Done.